data/css/course.scss

```diff
@@ -0,0 +1,187 @@
+@import "project-setup";
+@import "compass";
+
+/* === Colors and Fonts (Overrides) === */
+
+$brand-color: #005a71 !default;
+$primary-medium-color: #12282e !default;
+$primary-light-color: #cc532b !default;
+
+$accent-light-color: #f6f6f6 !default;
+$accent-medium-color: #ebb010 !default;
+$accent-dark-color: #5b700e  !default;
+$highlight-color: #f5d888  !default;
+
+$background-color: #e9e9e3 !default;
+
+$base-color: #0e82c5 !default;
+$base-color-shade: #0f6b9f !default;
+
+$body-font: Roboto !default;
+$title-font: Roboto !default;
+$display-font: Roboto !default;
+$context-font: Roboto !default;
+$support-font: 'Montserrat' !default;
+
+/* === Colors and Fonts === */
+
+@import 'assets';
```

Comment:
If https://phabricator.endlessm.com/T13798 lands this will need to be reworked to build off our default theme.